src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,99 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[],
+  personSlug: string,
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, personSlug }) => {
+  if (!people) {
+    return (
+      <p data-cy="noPeopleMessage">
+        There are no people on the server
+      </p>
+    );
+  }
+
+  const peopleWithParents = people.map(human => ({
+    ...human,
+    mother: people.find(person => person.name === human.motherName) || null,
+    father: people.find(person => person.name === human.fatherName) || null,
+  }));
```

Comment:
I think would be better to cover your function with useMemo() and put the bode of function to utils